server/conf/application.dev.conf

```diff
@@ -19,4 +19,6 @@ play.filters {
     cookie.secure = false
   }
   disabled += filters.HSTSFilter
+
+  gzip: null
```

Comment:
Change this to `disabled += "play.filters.gzip.GzipFilter"`

Reply:
@clouser2 makes a good argument for leaving gzip on in dev, I rescind my request for this

Reply:
SG